packages/opc-back-to-top/test/opc-back-to-top.test.ts

```diff
@@ -0,0 +1,35 @@
+import { LitElement } from 'lit-element';
+import { BackToTop } from '../src/opc-back-to-top';
+
+describe('opc-back-to-top', () => {
+
+    const OPC_COMPONENT = 'opc-back-to-top';
+    const ELEMENT_ID = 'opc-back-to-top';
+    let buttonElement: BackToTop;
+
+    beforeEach(() => {
+        buttonElement = window.document.createElement(OPC_COMPONENT) as BackToTop;
+        document.body.appendChild(buttonElement);
+    });
+
+    afterEach(() => {
+       document.querySelector(OPC_COMPONENT).remove();
+    });
+
+    it('is defined', async () => {
+        expect(buttonElement).toBeDefined();
```

Comment:
fix your indentation

Reply:
I followed select then right click then format document, is that same you follow to format your code ?